addon/components/transition/child.hbs

```diff
@@ -0,0 +1,20 @@
+<Transition
+  @show={{@show}}
+  @unmount={{@unmount}}
+  @tagName={{@tagName}}
+  @enter={{@enter}}
+  @enterFrom={{@enterFrom}}
+  @enterTo={{@enterTo}}
+  @leave={{@leave}}
+  @leaveFrom={{@leaveFrom}}
+  @leaveTo={{@leaveTo}}
+  @beforeEnter={{@beforeEnter}}
+  @afterEnter={{@afterEnter}}
+  @beforeLeave={{@beforeLeave}}
+  @afterLeave={{@afterLeave}}
```

Comment:
I wish there was a better way to pass along all of the arguments, but since there isn't, we need to manually forward all of the arguments to the inner component!